mm2src/coins/tendermint/tendermint_token.rs

```diff
@@ -493,7 +493,7 @@ impl MmCoin for TendermintToken {
         let coin_conf = crate::coin_conf(ctx, self.ticker());
         let wallet_only_conf = coin_conf["wallet_only"].as_bool().unwrap_or(false);
 
-        wallet_only_conf || self.platform_coin.is_keplr_from_ledger
+        wallet_only_conf || self.platform_coin.is_ledger_connection()
```

Comment:
These don't seem to be equal, why did we change this?

Reply:
what do you? I made an helper function since tendermint coin can be activated via WalletConnect via Ledger
https://github.com/KomodoPlatform/komodo-defi-framework/blob/59db165d9fd99e8cb98ac30e33dd7a691b40e89d/mm2src/coins/tendermint/tendermint_coin.rs#L2212-L2217

Reply:
We had some issues using Keplr with Ledger, but I don’t remember if it was entirely due to the Ledger itself even without Keplr.

Reply:
`WcLedger` can only be activated if a user connects via WalletConnect using Keplr mobile wallet.
So the issue should be the same

Reply:
I remember the main reason I did that. It was due to limitations in Ledger's transaction format policy. So it's not just about Keplr Ledger; it's a limitation with Ledger itself. Resolving.